docs/60_initialize_vendors.md

```diff
@@ -0,0 +1,425 @@
+# Initialize Vendor Accounts
+
+The DANDI ecosystem relies on vendor services to operate.  So first you will need to set up accounts with the following vendors:
+
+• **Heroku**: Provisions the API componenents
+• **AWS**: Provides storage buckets, as well as domain management, for resources across the DANDI ecosystem. As well as the services (Kubernetes, etc.) for deploying the JupyterHub.
+• **GitHub**: Serves as the authentication provider for accounts across the DANDI ecosystem
+• **Terraform Cloud**: Manages provisioned resources across cloud vendors in a version-controlled manner. 
+• **Netlify**: Deploys production frontend build, as well as staging previews to assist with frontend development
+• **Sentry**: Provides observability and monitoring for API events
+
+Some services are not yet integrated within the main infrastructure:
+
+• https://github.com/dandi/backups2datalad - to populate/update https://github.com/dandi/dandisets, https://github.com/dandisets, and https://github.com/dandizarrs/
+
+• TODO: heroku logs
+
+• TODO: aws s3 access stats dump
+
+• TODO: con/tinuous dumps of CI logs 
+
+• TODO: zarr manifests generation (ATM not on drogon even)
+
+• TODO: access stats analysis/plots (yet to be finished/cron deployed)
```

Comment:
should these remain as TODOs?

Reply:
I have filed issues in this repo.  These TODOs can now be removed from this doc.